### DISCUSSION TITLE
Alfven speed

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#49] [General]
For the mass limits we assume that the neutral density is a factor of  $c_s^2/v_A^2$, where $c_s$ is the ionized sound speed and $v_A$ is the alfven speed. For $v_A$ we assume a speed between 1 and 3 km/s.

On Jiménez - Hernández in #19 they calculated that the nebulous planetary was formed 11.8kyr ago, if we take that the mass loss is constant ($8\times10^5M_{/circ}/yr$) all this time, we have that $\dot{M}\times11.8kyr\approx.944M_\circ$.

Now, for $v_A=3km/s$ we have a total mass in the globules of $\approx.32M_\circ$ and for $v_A=2km/s$ we have a total mass of $\approx.72M_\circ$.

Can we say something about  $v_A$ with this?

_for_ $v_A=1km/s$ _we have a total mass of_ $2.9M_\circ$

**Comment by will-henney** [comment]
Yes this is an interesting comparison to make. 

However, the integrated stellar wind mass that you calculate is for the WR wind alone. It is possible that most of the mass of the nebula was ejected in an earlier Red Supergiant or LBV phase in which the Mdot was higher.

We should also compare with the total ionized gas mass of the nebula, which can be derived from the H alpha flux and the rms density.
